docs/get-started/getting-started-alluxio.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: page
+title: Alluxio
+nav_order: 5
+parent: Getting-Started
+---
+# Get Started with RAPIDS with Alluxio
+
+Alluxio is a data orchestration platform brings your data closer to compute across clusters,
+regions, clouds, and countries. This guide will go through how to set up the RAPIDS Accelerator
+for Apache Spark 3.0 with Alluxio and on premise cluster.
+
+## Prerequisites
+
+Assuming that user has successfully run the RAPIDS Accelerator with on premise cluster according
+to [this doc](getting-started-on-prem.md)
+
+This guide takes Yarn cluster as an example and assumes that there are 2 datanodes and 1 namenode in Yarn cluster, and their hostnames are respectively
+
+``` json
+namenode_hostname
+datanode1_hostname
+datanode2_hostname
+```
+
+## Alluxio setup
+
+It is recommending to deploy Alluxio workers in each datanode and Alluxio master in namenode.
+
+1. download the latest Alluxio version alluxio-${LATEST}-bin.tar.gz from [alluxio website](https://www.alluxio.io/download/)
+2. copy `alluxio-${LATEST}-bin.tar.gz` to all the datanodes and namenode
+3. extract `alluxio-${LATEST}-bin.tar.gz` to `/opt/alluxio-${LATEST}` in datanode and namenode
+   ``` shell
+   tar xvf alluxio-${LATEST}-bin.tar.gz -C /opt
+   ```
+4. configure alluxio
+    - Alluxio master configuration
+
+   add below recommending configuration in `/opt/alluxio-${LATEST}/conf/alluxio-site.properties`
+   ``` xml
+   alluxio.master.hostname=namenode_hostname
+
+   ######### worker properties #########
+   ## configure async cache manager
+   alluxio.worker.network.async.cache.manager.threads.max=64
+   alluxio.worker.network.async.cache.manager.queue.max=51200
+   alluxio.worker.tieredstore.levels=1
+   alluxio.worker.tieredstore.level0.alias=SSD
+   alluxio.worker.tieredstore.level0.dirs.mediumtype=SSD
+   alluxio.worker.tieredstore.level0.dirs.path=/YOUR_CACHE_DIR
+   alluxio.worker.tieredstore.level0.dirs.quota=/YOUR_CACHE_QUOTA
+
+   # enable domain socket
+   alluxio.worker.data.server.domain.socket.address=/YOUR_DOMAIN_SOCKET_PATH
+   alluxio.worker.data.server.domain.socket.as.uuid=true
+   ####################################
+
+   ######### User properties #########
+   ## enable circuit
+   alluxio.user.short.circuit.preferred=true
+   alluxio.user.short.circuit.enabled=true
+   ## use LocalFirstPolicy
+   alluxio.user.ufs.block.read.location.policy=alluxio.client.block.policy.LocalFirstPolicy
+   ####################################
+
+   # configure s3
+   alluxio.underfs.s3.endpoint=/S3_ENDPOINT
```

Comment:
state what S3_ENDPOINT should be.

Reply:
added a link for this.

Reply:
I assume you mean section Mount an existing S3 bucket to Alluxio.?  I was looking for reference to S3_ENDPOINT and didn't see it. I think we should put a comment here that says see that section for what S3_ENDPOINT means.

I've seen many people read a section like this, copy it and paste and then don't read further, so want to make sure its clear t user they have to do something with this and its easy to find what.